setup.py

```diff
@@ -9,6 +10,13 @@
 # globus-sdk does not support python versions older than 2.7 #
 ##############################################################""")
 
+# warn on older/untested python3s
+# it's not disallowed, but it could be an issue for some people
+if sys.version_info > (3,) and sys.version_info < (3, 4):
+    warnings.warn(
+        'Installing globus-sdk on Python 3 versions older than 3.4 '
+        'may result in degraded functionality or even errors.')
```

Comment:
Part of me wants this warning to be a little louder, but I guess people stuck on EOL python already have enough warnings yelling at them.